devicetypes/stelpro/stelpro-ki-thermostat.src/stelpro-ki-thermostat.groovy

```diff
@@ -49,8 +49,14 @@ metadata {
 	simulator { }
 
 	preferences {
-		input("heatdetails", "enum", title: "Do you want a detailed operating state notification?", options: ["No", "Yes"], defaultValue: "No", required: true, displayDuringSetup: true)
-		input("zipcode", "text", title: "ZipCode (Outdoor Temperature)", description: "[Do not use space](Blank = No Forecast)")
+		section {
+			input("heatdetails", "enum", title: "Do you want a detailed operating state notification?", options: ["No", "Yes"], defaultValue: "No", required: true, displayDuringSetup: true)
+		}
+		section {
+			input title: "Outdoor Temperature", description: "To get the current outdoor temperature to display on your thermostat enter a zipcode below.", displayDuringSetup: false, type: "paragraph", element: "paragraph"
+			input title: "", description: "Only U.S. based zipcodes are supported by the weather provider. However, if you are outside of the U.S. then enter your zipcode below and make sure that your SmartThings location has a Geolocation configured.", displayDuringSetup: false, type: "paragraph", element: "paragraph"
```

Comment:
I would say "Only US ZIP codes are supported by the weather provider. However, if you are outside the US, you can enter your _postal code_ to make sure that your SmartThings location supports other geolocation services."

Reply:
I think @stelpro 's suggestion above might be a bit better and possibly result in less confusion. I'm looking at how to make sure that the "SmartThings hub has its geolocation enabled" part makes sense. (I need to play with the configuration setting in the new ST app a little more to see what wording it uses to best be generic for both.)